index.html

```diff
@@ -269,24 +276,9 @@
 						<p><span class="fragment">Assign pull request to developer; review</span></p>
 						<p><span class="fragment">Both positive: merge the pull request</span></p>
 					</section>
-				</section>
-
-				<section>
-					<section>
-						<h1>Improving openness of our own projects</h1>
-					</section>
-					<section>
-						<h1>TODO</h1>
-					</section>
-				</section>
-				
-				<section>
-					<section>
-						<h1>GitHub integrations</h1>
-					</section>
 
 					<section>
-						<h1>Board</h1>
+						<h1>Process visualization</h1>
```

Comment:
I suggest to shortly show the Waffle screen shot as entry slide so that people have a clue what this is about. Then the bullet points. I would modify them though as it currently is not that easy to read. My proposal:

```
<p><span class="fragment">To get a unified view of a team's issues from different repos we use a Kanban board</span></p>
<p><span class="fragment">A Kanban board visualizes the development workflow and the current status of each issue</span></p>
<p><span class="fragment">Waffle is Kanban board with GitHub integration:<br/> every issue is a card, columns are associated with labels</span></p>
<p><span class="fragment">(GitHub now has an integrated board, but for now it's single-repo)</span></p>
```